geomstats/backend/numpy.py

```diff
@@ -290,3 +290,11 @@ def nonzero(x):
 
 def copy(x):
     return np.copy(x)
+
+
+def sign(*args, **kwargs):
+    return np.sign(*args, **kwargs)
+
+
+def mean(x, axis=None):
+    return np.mean(x, axis)
```

Comment:
You might not need this. If we want to stay "full geomstats", the mean of vectors, which is a mean in a euclidean space, should be written as:
`euclidean_space.metric.mean`.

Reply:
In a new commit it's not used anymore, so we can remove it

Reply:
mean() method removed from all backends